src/static/css/main.less

```diff
@@ -81,6 +81,16 @@ body {
     overflow-wrap: break-word;
 }
 
+blockquote {
+
+  border-left: 0.25em solid @gray;
+  color: @dark-gray;
+  padding: 0.5em 1em 0.25em 1em;
+
+  &_p {
+    display: inline;
+  }
```

Comment:
Is this intended to do something in the future, but isn't needed for this content?